src/test/java/io/axoniq/demo/university/faculty/write/subscribestudent/SubscribeStudentToCourseTest.java

```diff
@@ -1,132 +1,108 @@
 package io.axoniq.demo.university.faculty.write.subscribestudent;
 
-import io.axoniq.demo.university.UniversityAxonApplication;
+import io.axoniq.demo.university.UniversityApplicationTest;
 import io.axoniq.demo.university.faculty.events.CourseCreated;
 import io.axoniq.demo.university.faculty.events.StudentEnrolledInFaculty;
 import io.axoniq.demo.university.faculty.events.StudentSubscribedToCourse;
 import io.axoniq.demo.university.shared.ids.CourseId;
 import io.axoniq.demo.university.shared.ids.StudentId;
-import org.axonframework.test.fixture.AxonTestFixture;
-import org.junit.jupiter.api.*;
+import org.axonframework.eventsourcing.configuration.EventSourcingConfigurer;
+import org.junit.jupiter.api.Test;
 
-import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
-class SubscribeStudentToCourseTest {
+public class SubscribeStudentToCourseTest extends UniversityApplicationTest {
```

Comment:
They don't have to be `public`...right?
```suggestion
class SubscribeStudentToCourseTest extends UniversityApplicationTest {
```